lib/tasks/find_bad_ucf.rake

```diff
@@ -1,15 +1,15 @@
 namespace :freereg do
 
   desc "Rebuild UCF and deploy"
-  task :rebuild_ucf => [:environment] do
+  task :find_bad_ucf => [:environment] do
     #require '#{Rails.root}/lib/refresh_ucf_list'
     require 'refresh_ucf_list'
     # Print the time before start the process
     start_time = Time.now
     p "Starting at #{start_time}"
 
     # Call the RefreshUcfList library class file with passing the model name as parameter
-    entries = RefreshUcfList.new(Freereg1CsvEntry)
+    entries = RefreshUcfList.new(Freereg1CsvEntry, "C:/Work/FreeUKGen/MyopicVicar/scrpt/refresh_ucf")
```

Comment:
Let's default this to `File.join(Rails.root, 'script', 'refresh_ucf')` so that it will work without parameters on other machines.